src/Mapping/MappedEventSubscriber.php

```diff
@@ -269,4 +282,13 @@ private function getDefaultAnnotationReader(): Reader
 
         return self::$defaultAnnotationReader;
     }
+
+    private function getCacheItemPool(): CacheItemPoolInterface
+    {
+        if (null === $this->cacheItemPool) {
+            $this->cacheItemPool = new ArrayAdapter();
```

Comment:
Maybe we could make this assignment by default at `__construct()`. Is there a performance gain making it lazy here?